onnxscript/tests/function_libs/torch_aten/extra_opinfo.py

```diff
@@ -0,0 +1,108 @@
+"""
+Test data for aten operators which don't exist in PyTorch file:
+pytorch/torch/testing/_internal/common_methods_invocations.py.
+"""
+
```

Comment:
Should we add 

`__all__ = ["OP_DB"]`

?

Reply:
Not necessary now. And showing "extra_opinfo.OP_DB" in the code probably more readable.